bin/generate.sh

```diff
@@ -68,4 +68,13 @@ function generate_book() {
     echo 'scoreboard players enable @s uhcOpt' >> "$output_file"
 }
 
+function delete_file() {
+    local filename="$1"
+    if which trash; then
```

Comment:
> `if which trash; then`
`which` is not POSIX compliant. It might work, I guess Hench will have to test that one out.

In my experience anyway, `which` is not defined by a standard. This means different operating systems may implement it differently, which can lead to unpredictable behaviour.

May have to look into `command` or `type` instead

Reply:
The joy of trying to do cross-platform shell work. I'll take a look.

Reply:
I'm not overly concerned. If it works on Linux and Mac then that's good enough for me.

Reply:
Yeah fair enough, just wanted to make sure it didn't go under the radar and suddenly have it work for one and not the other hahah

Seems to work fine on Ubuntu WSL for what it's worth